client/src/js/services/authinjector.js

```diff
@@ -1,10 +1,14 @@
 angular.module('bhima.services')
-.factory('AuthInjectorFactory', ['$rootScope','$q',  '$location', 'SessionService', AuthInjector]);
+.factory('AuthInjectorFactory', AuthInjectorFactory);
 
-function AuthInjector($rootScope, $q, $location, Session) {
+AuthInjectorFactory.$inject = [
+  '$rootScope','$q',  '$location', 'SessionService'
+];
+
+function AuthInjectorFactory($rootScope, $q, $location, Session) {
   return {
     responseError : function (res) {
-      console.log('[INTERCEPTOR] Received HTTP error', res.status, res.data);
+      console.log('[INTERCEPTOR] Received HTTP error', res.status);
```

Comment:
It is helpful to know the error code that came back in development.  It should be removed in production code though.